examples/extended/grails-app/i18n/messages.properties

```diff
@@ -54,3 +54,7 @@ typeMismatch.java.lang.Short=Property {0} must be a valid number
 typeMismatch.java.math.BigDecimal=Property {0} must be a valid number
 typeMismatch.java.math.BigInteger=Property {0} must be a valid number
 typeMismatch=Property {0} is type-mismatched
+
+#Example of how to use the label for extra validation
```

Comment:
Would it be better if the comment read as follows?:

```
# Example of how to use the labelMessage for extra validation
```

Reply:
fixed in next commit